src/plugin-update/operation/updatework.cpp

```diff
@@ -1842,13 +1842,13 @@ CanExitTestingChannelStatus UpdateWorker::checkCanExitTestingChannelDialog()
     connect(watcher,
             &QFutureWatcher<CanExitTestingChannelStatus>::finished,
             this,
-            [watcher, dialog, &wantexit] {
+            [watcher, dialog, &wantexit, label] {
                 watcher->deleteLater();
                 auto result = watcher->result();
                 if (result == CanExitTestingChannelStatus::CheckError) {
                     wantexit = result;
-                    dialog->close();
-                    return;
+                    label->setText(tr("Waring, some pacakge in internal channel, do you still want "
```

Comment:
```suggestion
                    label->setText(tr("Warning, some pacakges in internal channel, do you still want "
```

对一下 v20 的文案与产品行为，文案由产品或翻译人员提供。